_common.sh

```diff
@@ -14,3 +14,11 @@ find_images () {
         grep "^${USERNAME}/racket:[[:digit:]]" | \
         sort;
 }
+
+find_test_images () {
+    # Version 6.0 is ignored during test runs because its openssl
+    # bindings are broken.
+    #
+    # xref: https://github.com/jackfirth/racket-docker/issues/35
+    find_images | grep -v "racket:6.0"
```

Comment:
What's the `-v` flag do? Is it negation?

Also, is there a longer alias for that flag we could use instead of the one-letter shorthand?

Reply:
Yes, it inverts the match. It looks like there's also `--invert-match` so I'll use that instead.